### ISSUE TITLE
Use global defaults if they exist upon invocation

### ISSUE THREAD

**Issue by niftylettuce** [limonte/sweetalert2#763]
Being able to define `window._swalDefaults` before `swal` is loaded in order to define defaults for `swal` on a page would be nice.

Perhaps we could add a line such as:

```js
swal.setDefaults(window._swalDefaults || {});
```

Or build this in to use `window._swalDefaults`?

```html
<script>window._swalDefaults = { ... };</script>
<script src="//some.com/path/to/sweetalert2.js"></script>
```

Not sure if there's some other way to do this without having to write the above line manually.

It'd be nice to have this for [Lad](https://lad.js.org) to simplify things

**Comment by niftylettuce** [comment]
@limonte I sent a pull request, can you please accept/merge/release new version to npm and ping me back?  thanks! ❤️

**Comment by limonte** [comment]
Merged and released in v7.1.0, https://jsfiddle.net/ad3quksn/263/
